Update templates.md (#223)
gofiber/docs: guide/templates.md

})
```

The `Render` method is linked to the [**ctx.Render\(\)**](templates.md) function that accepts a template name and binding data. It will use global layout if layout is not being defined in `Render` function.

> If the App option PassLocalsToViews is enabled. All locals set using ctx.Locals(key, value) Will passed to the template.

Reviewer: ```suggestion
If the Fiber config option `PassLocalsToViews` is enabled, then all locals set using `ctx.Locals(key, value)` will be passed to the template.
```
